### PR TITLE
Add soft delete message

### PR DESCRIPTION
**Pull request by vikram-canopy**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]()

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: dist/consolidated.json [hunk 1]
```diff
@@ -22667,7 +22674,8 @@
                         "description": "The role of this user in the API",
                         "example": 1
                       }
-                    }
+                    },
+                    "description": "The API User who authored the note"
```
Comment (vikram-canopy) [inline dist/consolidated.json:22678]: not sure why these deltas appear @ps2-controller , when I run `yarn build` I see `dist/consolidated.json is valid` in the output. But I see it was already updated in this commit in master: https://github.com/canopy1/api-docs/commit/095d2e154edd9d63d43c2c7275ae07a5f648d7c3

File: dist/consolidated.json [hunk 2]
```diff
@@ -5270,7 +5270,15 @@
         ],
         "responses": {
           "200": {
-            "description": "Account soft-deleted."
+            "description": "Account soft-deleted.",
+            "content": {
```
Comment (ps2-controller) [inline dist/consolidated.json:5274]: @vikram-canopy --- no objections to adding this text/plain content, but in general, we use the pattern "JSON" -- 
{
  "description": "Account deleted successfully." 
}

thoughts on moving in one direction or another?

Comment (vikram-canopy) [reply dist/consolidated.json:5274]: This is an artifact of me not understanding openAPI spec and trying to have a text resp with just the message. I converted to a JSON schema based approach later and didn't update the content type. Good catch

**Comment by vikram-canopy** [comment]
as discussed w/ @ps2-controller , these changes are unnecessary. We can already return a 'description' text field as per the existing api spec. Closing

**Pull request closed without merging**